release_notes_generator/utils/pull_request_utils.py

```
@@ -65,8 +67,12 @@ def get_issues_for_pr(pull_number: int) -> list[int]:
 
     response = requests.post(github_api_url, json={"query": query}, headers=headers, verify=False, timeout=10)
```

Ruff (0.13.1)

Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)